### PR TITLE
Preserve unsigned file

### PR DESCRIPTION
**Pull request by TravisEz13**
Preserve unsigned file
  - Also, add a malware scanning template and readme example.

### CODE REVIEW

File: EsrpScan.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+parameters:
```
Comment (adityapatwardhan) [inline EsrpScan.yml:1]: Can you add this template to assembly-module and script-module templates?

File: EsrpScan.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  - name: "scanPath"
+    default: "$(System.ArtifactsDirectory)"
```
Comment (adityapatwardhan) [inline EsrpScan.yml:3]: Maybe change this to `$(Pipeline.Workspace)`

File: EsrpScan.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  - name: "pattern"
+    default: "*.dll,*.exe"
+
+steps:
+- task: SFP.build-tasks.custom-build-task-2.EsrpMalwareScanning@1
+  displayName: 'Malware Scanning'
+  inputs:
+    ConnectedServiceName: pwshEsrpScanning
```
Comment (adityapatwardhan) [inline EsrpScan.yml:11]: This would need a new service connection in each project right?

File: EsrpScan.yml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    FolderPath: ${{ parameters.scanPath }}
+    Pattern: ${{ parameters.pattern }}
+    UseMinimatch: true
+    SessionTimeout: 30
+
+
```
Comment (adityapatwardhan) [inline EsrpScan.yml:17]: extra lines

**Review by adityapatwardhan — CHANGES_REQUESTED** [review]

File: EsrpScan.yml (hunk 3 above)
Comment (TravisEz13) [reply EsrpScan.yml:11]: If you use it, it does.  The service connection type is different.

File: EsrpScan.yml (hunk 2 above)
Comment (TravisEz13) [reply EsrpScan.yml:3]:
```suggestion
    default: "$(Pipeline.Workspace)"
```

File: EsrpScan.yml (hunk 1 above)
Comment (TravisEz13) [reply EsrpScan.yml:1]: I don't understand the ask here.

Comment (TravisEz13) [reply EsrpScan.yml:1]: This does not fit in a standard template.  It should only be used in custom designed pipelines that use multiple machines like PS/PS

**Review by adityapatwardhan — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -141,3 +141,25 @@ This example signs `dll` `psd1` and `psm1` files recursively, using minimatch.
             **\*.nupkg
         useMinimatch: true
 ```
+
+## ESRP Malware Scanning Template Overview
+
+** Requires on-boarding, see the wiki in the internal PowerShell Maintainers teams channel **
+
+Details can be found in the PowerShell Maintainers teams channel's Wiki tab.
```
Comment (TravisEz13) [inline README.md:149]:
```suggestion
Details can be found in the PowerShell Maintainers teams channel's Wiki tab.

This should be use in multi-Job builds when you are uploading unsigned binaries.
Files are automatically scanned on signing,
scanning on each upload will allow us to detect when any malware was introduced.
```

**Pull request merged**